bluesky/spec_api.py

```diff
@@ -267,8 +282,14 @@ def a2scan(*args, time=None, md=None):
         plan_stack.append(plan)
     return plan_stack
 
+
 # This implementation works for *all* dimensions, but we follow SPEC naming.
-a3scan = a2scan
+def a3scan(*args, time=None, md=None):
```

Comment:
planify is a work-around to make context managers work.  That they do is make a list of plans that should be run in sequence, `planify` just call `pchain` in the return value of the wrapped function.